src/transformers/models/flaubert/tokenization_flaubert.py

```diff
@@ -213,8 +213,6 @@ class FlaubertTokenizer(PreTrainedTokenizer):
             Dictionary mapping languages string identifiers to their IDs.
         id2lang (`Dict[int, str]`, *optional*):
             Dictionary mapping language IDs to their string identifiers.
-        do_lowercase_and_remove_accent (`bool`, *optional*, defaults to `True`):
-            Whether to lowercase and remove accents when tokenizing.
```

Comment:
I also removed this from the argument list in the doc. Hope this is a good decision :-)

Reply:
Yes, thanks!